packages/runtime/container-runtime/src/test/runningSummarizer.spec.ts

```diff
@@ -42,7 +42,7 @@ describe("Runtime", () => {
             const summaryConfig: ISummaryConfiguration = {
                 idleTime: 5000, // 5 sec (idle)
                 maxTime: 5000 * 12, // 1 min (active)
-                maxOps: 1000, // 1k ops (active)
```

Comment:
I'd not touch tests at all - they should use whatever they want (and ideally - different settings from defaults)

Reply:
For this one it does not really matter but for the summaryManager, we use the maxops to validate the summary behavior.